locales/da/messages.json

```diff
@@ -2371,7 +2371,7 @@
         "description": "Help text to AIRMODE mode"
     },
     "auxiliaryHelpMode_ALTHOLD": {
-        "message": "Højde tilstand er for nye piloter, den sikreste manuelle tilstand - uden GPS. Det er ligesom Stabil tilstand, men her låses dronens højde når pinde slippes",
+        "message": "Højde tilstand er for nye piloter, den sikreste manuelle tilstand - uden GPS. Det er ligesom Stabil (?) tilstand, men her låses fartøjets højde når pinde slippes",
```

Comment:
- why the `(?)` ?
- `locals/en/messages.json` contains the word `vehicle` here, therefore it was translated to `fartøjets` (en: vessel) -- i feel like the english should be `aircraft`, not `vehicle`.

Reply:
This must be amended on crowdin.

Reply:
i will make PR for `en` with other verbiage modifications. Can we discard this file? via 
![image](https://github.com/user-attachments/assets/dc7b3406-c98e-49e2-8864-0dcb03e6abf2)

Reply:
@McGiverGim please advise

Reply:
If we change the EN file, all the other languages will be discarded. If not, we need to modify the translation in Crowdin and approve it.